pkg/controllers/proxy/network_services_controller.go

```diff
@@ -445,7 +445,9 @@ func (nsc *NetworkServicesController) doSync() error {
 
 	if nsc.MetricsEnabled {
 		err = nsc.publishMetrics(nsc.serviceMap)
-		glog.Errorf("Error publishing metrics: %s", err.Error())
+		if err != nil {
+			glog.Errorf("Error publishing metrics: %v", err)
```

Comment:
Following the style of the above return, I would pull the return statement into the `if` expression as well. I get that it doesn't make a practical difference now, but it will keep us from getting caught later if we add more to the `doSync()` method.